marconi-chain-index/test/Integration.hs

```diff
@@ -145,11 +145,11 @@ testIndex = H.integration $ (liftIO TN.setDarwinTmpdir >>) $ HE.runFinallies $ H
       C.NoStakeAddress :: C.Address C.ShelleyAddr
 
   (tx1in, C.TxOut _ v _ _) <- do
-    utxo <- TN.findUTxOByAddress localNodeConnectInfo address
+    utxo <- TN.findUTxOByAddress @C.AlonzoEra localNodeConnectInfo address
```

Comment:
At some point, do we want this test to also run in Babbage era and up? Determining the era once at the start, or before, the test would be a step in the right direction. Same as the e2e tests do.

Reply:
@james-iohk From the e2e tests, Is [something like this](https://github.com/input-output-hk/plutus-apps/blob/main/plutus-e2e-tests/test/Spec/AlonzoFeatures.hs#L46) what you mean? I guess it is already possible to do it without changes to cardano-testnet package?

Reply:
Yes, can get the `CardanoEra` from `TestnetOptions` in use and pass it to any build/query function that needs it.